selfdrive/car/toyota/carcontroller.py

```diff
@@ -49,15 +61,15 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, hud_alert,
     # *** compute control surfaces ***
 
     # gas and brake
+    apply_gas = 0.
+    apply_accel = actuators.gas - actuators.brake
 
-    apply_gas = clip(actuators.gas, 0., 1.)
-
-    if CS.CP.enableGasInterceptor:
-      # send only negative accel if interceptor is detected. otherwise, send the regular value
+    if CS.CP.enableGasInterceptor and enabled and CS.out.vEgo < MIN_ACC_SPEED and \
+            apply_accel * CarControllerParams.ACCEL_SCALE > coast_accel(CS.out.vEgo):
       # +0.06 offset to reduce ABS pump usage when OP is engaged
-      apply_accel = 0.06 - actuators.brake
```

Comment:
We might need to still apply 0.06 offset only when applying brake, since it's a bit jerky for me. I'll test and update if it's better